driver/bpf/fillers.h

```diff
@@ -2995,12 +3002,13 @@ FILLER(sys_setns_e, true)
 
 FILLER(sys_unshare_e, true)
 {
-	unsigned long val;
-	u32 flags;
+	unsigned long val, v1, v2;
+	u32 flags = 0;
 	int res;
 
 	val = bpf_syscall_get_argument(data, 0);
-	flags = clone_flags_to_scap(val);
+	if (val > 1 || val < 0xffffffff - 1)
```

Comment:
Shouldn't this be an `&&`? Otherwise I think this condition will always be true.
```suggestion
	if (val > 1 && val < 0xffffffff - 1)
```